### PR TITLE
feat: load img support

### PR DESCRIPTION
**Pull request by gaurav274**

### CODE REVIEW

File: eva/catalog/catalog_manager.py [hunk 1]
```diff
@@ -388,50 +429,50 @@ def check_table_exists(self, database_name: str, table_name: str):
     def get_all_udf_entries(self):
         return self._udf_service.get_all_udfs()
 
-    def get_video_metadata_table(
+    def get_media_metainfo_table(
```
Comment (jarulraj) [inline eva/catalog/catalog_manager.py:432]: Maybe add a comment on the difference between `metainfo` and `metadata`.

File: eva/catalog/catalog_type.py [hunk 2]
```diff
@@ -22,6 +22,7 @@ class Dimension(IntEnum):
 class TableType(IntEnum):
     STRUCTURED_DATA = auto()
     VIDEO_DATA = auto()
+    IMAGE_DATA = auto()
```
Comment (jarulraj) [inline eva/catalog/catalog_type.py:26]: Do we need a very different approach to store AUDIO_DATA or PDF_DATA?

**Review by jarulraj — APPROVED** [review]

File: eva/catalog/catalog_manager.py (hunk 1 above)
Comment (gaurav274) [reply eva/catalog/catalog_manager.py:432]: The catalog's current naming convention is ineffective, and I am working on a new pull request to improve it.

File: eva/catalog/catalog_type.py (hunk 2 above)
Comment (gaurav274) [reply eva/catalog/catalog_type.py:26]: It should be similar

**Pull request merged**